.circleci/config.yml

```diff
@@ -2,7 +2,9 @@ version: 2
 jobs:
   validate:
     docker:
-      - image: circleci/golang:1.10
+      - image: circleci/golang:1.11
```

Comment:
Any reason not to use Go 1.12, where the module cache has been ironed out a little bit better.

Reply:
I think it's also a lot less work, given a lot of the places where you're explicitly opting into Go module support, 1.12 does automatically.

Reply:
@sargun I will give a try to `1.12` and remove the unnecessary flags.